tests/pass/lazy.glu

```diff
@@ -7,5 +7,8 @@ let { lazy, force } = import! std.lazy
 
 let l = lazy (\_ -> 123 + 57)
 
-assert_ieq (force (lazy (\_ -> 2))) 2 *> wrap ()
-    *> assert_ieq 180 (force l)
+let tests : Test () =
```

Comment:
These type annotations does not appear to be necessary (no need to remove them though). I will look into why importing ``Test` is necessary though, way way back that used to be necessary in a few cases so there may be some old code backfiring on implicits now. Even if it ends up being needed to shoudl at least give abetter error than it does now.